assets/js/src/navigation.js

```diff
@@ -76,7 +76,7 @@ function initEachNavToggleSubmenu( nav ) {
 			dropdown.appendChild( dropdownSymbol );
 
 			// Add before submenu.
-			submenu.parentNode.insertBefore( dropdown, submenu );
+			SUBMENUS[ i ].parentNode.insertBefore( dropdown, SUBMENUS[ i ]);
```

Comment:
Can you fix the spacing? `SUBMENUS[i].parentNode.insertBefore( dropdown, SUBMENUS[i] );`

Reply:
Done, but similar error reported by the build script.